### PR TITLE
Switch from flag to cobra for command-line parsing

### LINKED ISSUE #13: Tracker port should be configurable through commandline parameter
To make it consistent with other components.

### PR DESCRIPTION
**Pull request by yiranwang52**
- Switch from flag to cobra for command-line parsing.
- Add port argument to tracker (fix #13 ).
- Remove default value for agent registry port.
- Remove unused README and config files.

### CODE REVIEW

File: agent/cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+			start()
+		},
+	}
+)
+
+func Execute() {
+	rootCmd.Execute()
+}
+
+func start() {
```
Comment (codygibb) [inline agent/cmd/root.go]: Nit: `start` usually applies asynchronous behavior. This function blocks, and should instead be called `run`.

File: agent/cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+var (
+	peerIP            string
+	peerPort          int
+	agentServerPort   int
+	agentRegistryPort int
+	configFile        string
+	zone              string
+	krakenCluster     string
```
Comment (codygibb) [inline agent/cmd/root.go]: Maybe s/krakenCluster/cluster to stay consistent with the CLI flag name?

File: agent/cmd/root.go [hunk 3]
```diff
@@ -0,0 +1,188 @@
+package cmd
```
Comment (codygibb) [inline agent/cmd/root.go]: Maybe call this file `cmd.go` instead of `root.go`?

File: agent/cmd/root.go (hunk 1 above)
Comment (yiranwang52) [reply agent/cmd/root.go]: sure

File: agent/cmd/root.go (hunk 2 above)
Comment (yiranwang52) [reply agent/cmd/root.go]: in other components, cluster is used for something else and krakenCluster has to be used as parameter name. Just keeping it consistent here.

File: agent/cmd/root.go (hunk 3 above)
Comment (yiranwang52) [reply agent/cmd/root.go]: ok

File: agent/cmd/root.go (hunk 2 above)
Comment (codygibb) [reply agent/cmd/root.go]: Makes sense.

**Pull request merged**